src/app/table/table.component.ts

```diff
@@ -68,7 +76,7 @@ export class TableComponent implements OnInit {
 
     this.id = setInterval(() => {
       this.getRestItems();
-    }, 10000);
```

Comment:
check the time interval of polling

Reply:
I increases the time interval, because 10 sec is to early

Reply:
ok